grpc_server.go

```diff
@@ -86,11 +75,22 @@ func (s *server) Get(ctx context.Context, in *hegel.GetRequest) (*hegel.GetRespo
 }
 
 func (s *server) Subscribe(in *hegel.SubscribeRequest, stream hegel.Hegel_SubscribeServer) error {
-	active_subscription.Inc()
-	defer active_subscription.Dec()
+	metrics.TotalSubscriptions.Inc()
+	metrics.Subscriptions.WithLabelValues("initializing").Inc()
+	initError := func(err error) error {
+		s.log.Error(err)
```

Comment:
would be nice to have the ip address logged too once its been extracted from the request

Reply:
Good point